mm_cataloguer/generate_manifest.py

```diff
@@ -0,0 +1,19 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from modelmeta import DataFile, DataFileVariable, Ensemble, EnsembleDataFileVariables
+from datetime import datetime
+
+def list_filepaths(session, ensemble, since):
+    if ensemble == ["all"]:
+        ensemble = ["all_files"]
+    since = datetime.strptime(since, "%Y-%m-%d")
+    print(session.query(DataFile.filename).join(DataFileVariable)
```

Comment:
This would be gravy, but sometimes it's nice to give the option of outputting to a specific file, and then if the file isn't given use `sys.stdout` by default (which is the same as `print` with no arguments). This gives the user a bit more flexibility.